packages/react/src/TooltipV2/Tooltip.tsx

```diff
@@ -303,11 +303,10 @@ export const Tooltip = React.forwardRef(
               },
               onFocus: (event: React.FocusEvent) => {
                 // only show tooltip on :focus-visible, not on :focus
-                try {
-                  if (!event.target.matches(':focus-visible')) return
-                } catch (error) {
-                  // jsdom (jest) does not support `:focus-visible` yet and would throw an error
-                  // https://github.com/jsdom/jsdom/issues/3426
+                // Note: jsdom (jest) does not support `:focus-visible` yet
+                // https://github.com/jsdom/jsdom/issues/3426
+                if (!event.target.matches(':focus-visible') && !isJSDOM()) {
```

Comment:
If reading the line of `!event.target.matches(':focus-visible')` doesn't throw an error (looks like it doesn't since the tests are all green), we are good!